### ISSUE TITLE
README formatting messed up on PyPI

### ISSUE THREAD

**Issue by aaronreidsmith** [tgsmith61591/pmdarima#112]
The formatting for the README is getting screwed up when we release to PyPI ([example](https://pypi.org/project/pmdarima/)). This has only been an issue since `1.1.0`, [`1.0.0`'s formatting](https://pypi.org/project/pmdarima/1.0.0/) looks fine.

I suspect this has something to do with our Markdown being interpreted as RST. I will look into this

**Comment by aaronreidsmith** [comment]
@tgsmith61591 do we explicitly install `readme_renderer[md]` before distribution? Based on my local testing, I feel like that is the issue.

I know you made the `make` target, but it only relies on `bdist_wheel` (and by extension `requirements`), so `twine` doesn't appear to be checked for before testing the README.
 
Before `readme_renderer[md]`:
```
(pmdenv) asmith ~/Documents/projects/pmdarima (pypi-docs) $ twine check dist/*
/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/readme_renderer/markdown.py:38: UserWarning: Markdown renderers are not available. Install 'readme_render[md]' to enable Markdown rendering.
  warnings.warn(_EXTRA_WARNING)
/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/readme_renderer/markdown.py:50: UserWarning: Markdown renderers are not available. Install 'readme_render[md]' to enable Markdown rendering.
  warnings.warn(_EXTRA_WARNING)
Checking distribution dist/pmdarima-1.2.0.dev0-cp36-cp36m-macosx_10_7_x86_64.whl: Failed
The project's long_description has invalid markup which will not be rendered on PyPI. The following syntax errors were detected:
Checking distribution dist/pmdarima: InvalidDistribution: Unknown distribution format: 'pmdarima'
(pmdenv) asmith ~/Documents/projects/pmdarima (pypi-docs) $ twine check dist/*
/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/readme_renderer/markdown.py:38: UserWarning: Markdown renderers are not available. Install 'readme_render[md]' to enable Markdown rendering.
  warnings.warn(_EXTRA_WARNING)
/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/readme_renderer/markdown.py:50: UserWarning: Markdown renderers are not available. Install 'readme_render[md]' to enable Markdown rendering.
  warnings.warn(_EXTRA_WARNING)
Checking distribution dist/pmdarima-1.2.0.dev0-cp36-cp36m-macosx_10_7_x86_64.whl: Failed
The project's long_description has invalid markup which will not be rendered on PyPI. The following syntax errors were detected:
```

After `readme_renderer[md]`:
```
(pmdenv) asmith ~/Documents/projects/pmdarima (pypi-docs) $ twine check dist/*
Checking distribution dist/pmdarima-1.2.0.dev0-cp36-cp36m-macosx_10_7_x86_64.whl: Passed
```

**Comment by tgsmith61591** [comment]
Feel free to add a `pip install twine` and any other steps to that target!

**Comment by aaronreidsmith** [comment]
Fixed in #116
